src/main.scss

```diff
@@ -21,7 +80,9 @@ body {
   padding: 0;
 }
 
-html, body, .container {
```

Comment:
i dont like this rule :( convince me?

Reply:
Hard to discern, consistency with other scss files.

(I used to hate this rule too)